### DISCUSSION TITLE
Cannot find report file

### DISCUSSION THREAD

**Discussion by lehmannk** [fzyzcjy/flutter_convenient_test#349] [Q&A] [answered]
I have the toggle "SaveReport" switched ON but I cannot find a report file anywhere. I guess it is called "report.bin" as in the example? Is it possible to define the report location anywhere? Same goes for the CLI mode - with command `convenient_test_manager_dart.dart --enable-report-saver` no report file is produced.

**Comment by fzyzcjy** [comment, marked as answer]
According to https://github.com/fzyzcjy/flutter_convenient_test/blob/c24191d1298b1f969b2053b8b98891d58161cf15/packages/convenient_test_manager_dart/lib/services/report_saver_service.dart#L34, https://github.com/fzyzcjy/flutter_convenient_test/blob/c24191d1298b1f969b2053b8b98891d58161cf15/packages/convenient_test_manager_dart/lib/services/fs_service.dart#L11C31-L11C31, it is currently in `/tmp/ConvenientTest` (or whatever temporary directory your computer uses). Feel free to make a PR to allow this to be customized!
